src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -621,10 +609,16 @@ private FingerprintIdDetails createUnknownFP(final File samFile, final SAMRecord
                         retval.putAll(fingerprintVcf(f));
                     }
 
+                    synchronized (log) {
```

Comment:
you tell me...what's the right way to protect the log object from multiple threads using it at the same time, possibly putting it in an inconsistent state?

Reply:
Do you have reason to believe that the underlying log API is not thread safe?

if it's not thread safe, I don't think that wrapping individual calls like this will help.

Reply:
good point. I looked in the Logger class. it's already taken care of properly. thanks.